recipes/rb-yajl-ruby/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "yajl-ruby" %}
+{% set version = "1.4.1" %}
+
+package:
+  name: rb-{{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://rubygems.org/downloads/{{ name }}-{{ version }}.gem
+  sha256: 6e32e6e681c2861e99e7389ebcca70d56cb9357b223f09fb0857e2444ee1609b
+
+build:
+  number: 0
+  script:
+    - gem install -N -l -V --norc --ignore-dependencies {{ name }}-{{ version }}.gem
+    - gem unpack {{ name }}-{{ version }}.gem
+    - make -C {{ PREFIX }}/lib/ruby/gems/{{ ruby }}.0/gems/{{ name }}-{{ version }}/ext/yajl clean
```

Comment:
This should probable be in a build.sh script

Reply:
@scopatz, would it be okay to keep these here for the ruby packages? 
It makes reviewing these a lot simpler for me since the recipe ends up coming from the rb-example template in #9320, and these commands should mostly work on Windows when we get ruby building there.

Reply:
I think this is not great, though because it doesn't have `set -e`, though, and windows would be even worse for exiting after the first failure

Reply:
@sodre @scopatz @conda-forge/help-ruby what is the status on this issue? this has become the norm in all ruby compiled packages so far

Reply:
@scopatz, you bring a valid point about `set -e`. For the *unix* side, would you be okay with the syntax as such:
```yaml
script:
  - gem instal ...
    && gem unpack ...
    && make -C ... clean
```